ui/imodel-components-react/src/imodel-components-react/quantityformat/FormatType.tsx

```diff
@@ -35,7 +32,7 @@ interface FormatTypeSelectorProps extends CommonProps {
  * @alpha
  */
 function FormatTypeSelector(props: FormatTypeSelectorProps) {
-  const { type, onChange, ...otherProps } = props;
+  const { type, onChange } = props;
```

Comment:
Let's continue passing down other props for extensibility, let's name them `rest`.

```suggestion
  const { type, onChange, ...rest } = props;
```

Reply:
done here and across other components